src/cider/nrepl.clj

```diff
@@ -3,92 +3,123 @@
             [clojure.tools.nrepl.middleware.session :refer [session]]
             [clojure.tools.nrepl.middleware.pr-values :refer [pr-values]]
             [clojure.tools.nrepl.server :as nrepl-server]
+            [cider.nrepl.version :as version]
             [cider.nrepl.middleware.util.cljs :as cljs]
             [cider.nrepl.middleware.pprint :as pprint]
-            [cider.nrepl.middleware.version]
             [cider.nrepl.print-method]))
 
-(def DELAYS (atom nil))
+(def DELAYED-HANDLERS
```

Comment:
ALL_CAPS is unidiomatic Clojure. Any reason we're doing that here?

Reply:
It's good to differentiate between local and global non-function vars. It enhances readability. Any form of marking down is fine with me, but ALL-CAPS seems nicest to me. Some use `xyz_` for this purpose.

Reply:
Well, anything non-private is in effect global, so I'm fine with sticking with the standard notation. Anyways, that's something tiny.